index.html

```diff
@@ -4,32 +4,90 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <link rel="stylesheet" href="styles/modal.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <script
       src="https://kit.fontawesome.com/579b112cc6.js"
       crossorigin="anonymous"
     ></script>
-    <script src="index.js" defer></script>
+    <script src="js/index.js" defer></script>
 
     <title>My Portfolio</title>
   </head>
   <body>
+    <div class="modalView no-display">
+      <div class="modalContainer">
+        <img class="closeBtn" src="./images/menu_close.svg" alt="Close button" />
+        <h1 class="modalHeading">Project name goes here</h1>
+        <ul class="modalLanguage">
+          <li>HTML/CSS</li>
```

Comment:
- [ ] [OPTIONAL] I think it is better to dynamically load this content from javascript instead of hiding and unhiding the modal. Please remove these HTML code and load it from javascript when see project button is clicked.